src/openrct2/actions/RideEntranceExitRemoveAction.cpp

```diff
@@ -46,22 +46,20 @@ void RideEntranceExitRemoveAction::Serialise(DataSerialiser& stream)
 }
 
 static TileElement* FindEntranceElement(
-    const CoordsXY& loc, ride_id_t rideIndex, int32_t stationNum, int32_t entranceType, bool ghost)
+    const CoordsXY& loc, ride_id_t rideIndex, int32_t stationNum, int32_t entranceType, uint32_t flags)
```

Comment:
Is this change necessary?

Reply:
I wanted to get this a bit more consistent with the others so I opted to use the flags.

Reply:
Which others? 

If it does not need to know about flags, then I don't think it should receive them. If anything, the others should be modified if they never use any other flags besides `GAME_COMMAND_FLAG_GHOST`.

But I don't want to keep this PR from being merged just because of this, so this is fine for now.